### ISSUE TITLE
Missing CSS property for pre.sf-dump

### ISSUE THREAD

**Issue by Aybee** [contao/contao#2058]
**Affected version(s)**

Seen in 4.9.4

**Description**

There's a Rocksolid theme with this CSS rule.
```css
pre {
  background-image: linear-gradient(to bottom, #ececec 50%, #232323 50%);
}
```
This makes the output of `$this->dumpTemplateVars();` unreadable.

Following rule will correct this.
```css
pre.sf-dump {
  background-image: none;
}
```

**Comment by fritzmg** [comment]
Not sure this is something that should be fixed in Contao, since neither of these things originate directly from Contao.

**Comment by aschempp** [comment]
It should actually be fixed in Symfony the same way we fixed Contao: #1978  
@aybee to you want to create a pull request for Symfony or should I do that?

**Comment by Aybee** [comment]
Andreas, das wäre sehr nett, wenn du das machen könntest. Weiß nicht, ob ich da die richtige Stelle finden würde.
